docs/points/points_troubleshooting.rst

```diff
@@ -1,6 +1,15 @@
-.. vale off
-
 Points troubleshooting
```

Comment:
Do we still need to have this as a page heading? I think we do - not sure why it is removed here.

Reply:
Yes, we do but what was removed was the vale warning which wasn't needed